src/PrettierEditProvider.ts

```diff
@@ -92,7 +97,8 @@ async function format(
             parser: parser,
             semi: vscodeConfig.semi,
             useTabs: vscodeConfig.useTabs,
-        },
+            proseWrap: vscodeConfig.proseWrap,
+        } as PrettierConfig,
```

Comment:
Was there a problem without the cast?

Reply:
There was no type-checking on it before. Now it will be an error if incorrect properties are added